code/datums/antagonists/antag_datum.dm

```diff
@@ -0,0 +1,102 @@
+var/global/list/antagonists = list()
+
+/datum/antagonist
+	var/name = "Antagonist"
+	var/datum/mind/owner						//Mind that owns this datum
+	var/silent = FALSE							//Silent will prevent the gain/lose texts to show
+	var/can_coexist_with_others = TRUE			//Whether or not the person will be able to have more than one datum
+	var/list/typecache_datum_blacklist = list()	//List of datums this type can't coexist with
+	var/delete_on_mind_deletion = TRUE
+	var/antag_role //the antag type
+	var/replace_banned = TRUE //Should replace jobbaned player with ghosts if granted.
+	var/list/objectives = list()
+	var/antag_memory = ""//These will be removed with antag datum
+
+
+
+/datum/antagonist/New(datum/mind/new_owner)
+	antagonists += src
+	typecache_datum_blacklist = typecacheof(typecache_datum_blacklist)
+	if(new_owner)
+		owner = new_owner
+
+/datum/antagonist/Destroy()
+	antagonists -= src
```

Comment:
`GLOB.antagonists -= src`

Reply:
Yeah I didn't update this for me the globs hah